internal/resource/statefulset.go

```diff
@@ -436,12 +436,32 @@ func (builder *StatefulSetBuilder) podTemplateSpec(annotations, labels map[strin
 	}
 
 	tlsSpec := builder.Instance.Spec.TLS
-	if tlsSpec.SecretName != "" {
-		// add tls port
+	if builder.Instance.TLSEnabled() {
+		// add tls port, should we disable non tls ports here?
```

Comment:
Lets remove the comment here since we've already created an issue for this

Reply:
Makes sense, thanks for pointing that out! https://github.com/rabbitmq/cluster-operator/pull/451/commits/b737ca4a59b2e05ac2ba106efb485dbe6f603fcd